pallets/avn-proxy/src/tests/mock.rs

```diff
@@ -389,18 +388,19 @@ impl TestAccount {
         return ecdsa::Pair::from_seed(&self.seed)
     }
 
-    pub fn ethereum_ecdsa_sign(&self, message_to_sign: &[u8]) -> Signature {
-        let hashed_message = hash_with_ethereum_prefix(&hex::encode(message_to_sign)).unwrap();
-        return Signature::from(self.ecdsa_key_pair().sign(&hashed_message))
+    pub fn ethereum_ecdsa_sign(&self, _message_to_sign: &[u8]) -> Signature {
+        // TODO: Use a proper library to sign
```

Comment:
why not ethers.rs?

Reply:
Yeah, or any other that is easier to integrate.